Sources/Starknet/Data/Transaction/Transaction.swift

```diff
@@ -107,6 +107,122 @@ public struct StarknetInvokeTransaction: StarknetTransaction, Codable {
     }
 }
 
+public struct StarknetSequencerDeployAccountTransaction: StarknetSequencerTransaction, Equatable {
```

Comment:
Why differentiate between `StarknetSequencerTransaction` and `StarknetTransaction`? Can we just make `hash` an optional property?